### ISSUE TITLE
kissanime.ru

### ISSUE THREAD

**Issue by NatoBoram** [NanoAdblocker/NanoFilters#150]
### URLs where this issue occurs (Required)

http://kissanime.ru

### Screenshots (Required)

![image](https://user-images.githubusercontent.com/10495562/43372151-bdfc8702-936b-11e8-9a35-8b90e0e11a12.png)

### Screenshot of console (Required, press `F12` to open the console)

```
Loading failed for the <script> with source “https://www.googletagmanager.com/gtag/js?id=UA-1712467-32”. Episode-001:72
Loading failed for the <script> with source “http://native.propellerads.com/1?z=1827727&eid=p_1827727”. Episode-001:359
Loading failed for the <script> with source “http://native.propellerads.com/1?z=1731229&eid=p_1731229”. Episode-001:446
Loading failed for the <script> with source “http://native.propellerads.com/1?z=1733389&eid=p_1733389”. Episode-001:447
Content Security Policy: The page’s settings blocked the loading of a resource at self (“script-src”). Source: call to eval() or related function blocked by CSP.
Content Security Policy: The page’s settings blocked the loading of a resource at self (“script-src”). Source: (function(a,v,q,P){var c={safeWindow:{},.... 3UpQWmNa:1
Content Security Policy: The page’s settings blocked the loading of a resource at self (“script-src”). Source: try {
(function() {
	var cValue = 'noopF.... 3UpQWmNa:1
An iframe which has both allow-scripts and allow-same-origin for its sandbox attribute can remove its sandboxing. Episode-001
Loading failed for the <script> with source “http://ads.2mdnsys.com/qcas.php?sid=3&unit=5b4769e87b92b&did=42&zid=120&tm=1531406824&k=5b4769e87b970”. madads160.aspx:13
Loading failed for the <script> with source “https://go.oclasrv.com/apu.php?zoneid=847358”. 3UpQWmNa:115
Loading failed for the <script> with source “https://pushance.com/ntfc.php?zoneid=1543574”. 3UpQWmNa:130
Content Security Policy: The page’s settings blocked the loading of a resource at self (“script-src”). Source: onfocusin attribute on DIV element. 3UpQWmNa
Content Security Policy: The page’s settings blocked the loading of a resource at self (“script-src”). Source: 
  document.addEventListener('contextmen.... 3UpQWmNa:33
Content Security Policy: The page’s settings blocked the loading of a resource at self (“script-src”). Source: 
/*<![CDATA[/* */
  var _pop = _pop || [.... 3UpQWmNa:90
Content Security Policy: The page’s settings blocked the loading of a resource at self (“script-src”). Source: A8EE.y7=function (){return typeof A8EE.l.... 3UpQWmNa:115
Content Security Policy: The page’s settings blocked the loading of a resource at self (“script-src”). Source: onerror attribute on SCRIPT element. 3UpQWmNa
Content Security Policy: The page’s settings blocked the loading of a resource at self (“script-src”). Source: onload attribute on SCRIPT element. 3UpQWmNa
Content Security Policy: The page’s settings blocked the loading of a resource at https://go.oclasrv.com/apu.php?zoneid=847358 (“script-src”).
Content Security Policy: The page’s settings blocked the loading of a resource at self (“script-src”). Source:  (function (d, w, c) { (w[c] = w[c] || [.... 3UpQWmNa:117
Content Security Policy: The page’s settings blocked the loading of a resource at self (“script-src”). Source: A8EE.y7=function (){return typeof A8EE.l.... 3UpQWmNa:130
Content Security Policy: The page’s settings blocked the loading of a resource at self (“script-src”). Source: onerror attribute on SCRIPT element. 3UpQWmNa
Content Security Policy: The page’s settings blocked the loading of a resource at self (“script-src”). Source: onload attribute on SCRIPT element. 3UpQWmNa
Content Security Policy: The page’s settings blocked the loading of a resource at https://pushance.com/ntfc.php?zoneid=1543574 (“script-src”).
VIDEOJS: WARN: videojs.plugin() is deprecated; use videojs.registerPlugin() instead video.min.js:1:29532
Content Security Policy: The page’s settings blocked the loading of a resource at self (“script-src”). Source: 

videojs('videojs').ready(function() {
.... 3UpQWmNa:152
Content Security Policy: The page’s settings blocked the loading of a resource at self (“script-src”). Source: 
var t_run = 0;
var v_counter = 0;
$("#h.... 3UpQWmNa:217
An iframe which has both allow-scripts and allow-same-origin for its sandbox attribute can remove its sandboxing. Episode-001
The character encoding of a framed document was not declared. The document may appear different if viewed without the document framing it. pixfuture728.aspx
Loading failed for the <script> with source “http://ads.2mdnsys.com/qcas.php?sid=3&unit=5b4769e87b92b&did=42&zid=120&tm=1531406824&k=5b4769e87b970”. madads160.aspx:13 
```

### Describe the issue (Optional if obvious)

Video gets hidden after a few seconds of ads not loading

### Reproduction Steps (Optional if trivial)

1. Open kissanime.ru
2. Watch an anime. I can't link a direct page of an anime because of `GET` parameters.

After completing the captcha, the video will appear, then disappear after a few seconds.

### Anything else you believe to be useful (Optional)

It seems like the blocking is handled in JavaScript and could be circumvented using JavaScript. Blocking all JavaScript breaks the player.

#### Environment (Required)

- Operating System and Version

```bash
neofetch --off
nato@NatoUbuntu 
--------------- 
OS: Ubuntu 18.04.1 LTS x86_64 
Kernel: 4.15.0-29-generic 
Uptime: 2 hours, 18 mins 
Packages: 1919 
Shell: bash 4.4.19 
Resolution: 1920x1080 
DE: GNOME 3.28.2 
WM: GNOME Shell 
WM Theme: Adwaita 
Theme: Adwaita-dark [GTK2/3] 
Icons: Ubuntu-mono-dark [GTK2/3] 
Terminal: gnome-terminal 
CPU: Intel i5-3570K (4) @ 4.100GHz 
GPU: NVIDIA GeForce GTX 660 Ti 
Memory: 6262MiB / 7922MiB 
```

- Browser and Version: Firefox

![image](https://user-images.githubusercontent.com/10495562/43372223-812f8eae-936c-11e8-89a7-284aa7b7a080.png)

- Nano Adblocker Version: 

uBlock Origin

- Nano Defender Version (Optional if you are not using it): 

```
[Nano Adblocker]
! Title: Nano filters
! Expires: 1 days
! License: GPL-3.0
! Source: https://github.com/NanoAdblocker/NanoFilters/tree/master/NanoFiltersSource/NanoBase.txt
! This file is a compiled binary, do not modify
! All modifications will be overwritten on the next build
```

```
[Nano Adblocker]
! Title: Nano Defender Integration
! Expires: 3 days
! Cached: Sat, 28 Jul 2018 03:06:53 GMT
! License: GPL-3.0
! Source: https://github.com/jspenguin2017/uBlockProtector/tree/master/list
! This file is a compiled binary, do not modify
! All modifications will be overwritten on the next build
```

#### Your filter lists (Required)

* uBlock filters​​​​​
* uBlock filters – Annoyances​​​​​
* uBlock filters – Badware risks​
* uBlock filters – Experimental​
* uBlock filters – Privacy​​​​​
* uBlock filters – Resource abuse​​​
* uBlock filters – Unbreak
* Adblock Warning Removal List​
* Adguard Base Filters​
* Adguard Mobile Filters​
* EasyList​
* EasyPrivacy​
* Malvertising filter list by Disconnect​​​​​
* Malware Domain List​​​​​
* Malware domains​
* Spam404​
* Fanboy’s Annoyance List​
* Peter Lowe’s Ad and tracking server list​
* FRA: EasyList Liste FR​
* AakList (Anti-Adblock Killer)​​
* Beemoov Annoyances​​
* Nano Defender Integration​​
* Nano filters​​
* Nano filters - Whitelist​​

#### Your custom filters (Required if you have any)

Beemoov Annoyances​​ - https://gitlab.com/NatoBoram/Beemoov-Annoyances

**Comment by jspenguin2017** [comment]
Duplicate of https://github.com/jspenguin2017/uBlockProtector/issues/961

> I can't link a direct page of an anime because of `GET` parameters. 

That makes no sense. 

> ![image](https://user-images.githubusercontent.com/7283682/43372550-35b0876a-935f-11e8-8111-e3ead887b75d.png)

I mean the extension, not filters.
